src/lib.rs

```diff
@@ -97,7 +107,7 @@ pub fn nthash(seq: &[u8], ksize: usize) -> Vec<u64> {
 ///     use nthash::NtHashIterator;
 ///
 ///     let seq = b"ACTGC";
-///     let iter = NtHashIterator::new(seq, 3);
+///     let iter = NtHashIterator::new(seq, 3).unwrap();
 ///     let hashes: Vec<u64> = iter.collect();
 ///     assert_eq!(hashes,
 ///                vec![0x9b1eda9a185413ce, 0x9f6acfa2235b86fc, 0xd4a29bf149877c5c]);
```

Comment:
I would like to keep the docs `unwrap`-free, how about changing it to
```
/// ```
/// # use nthash::result::Result;
/// use nthash::NtHashIterator;
/// # fn main() -> Result<()> {
///     let seq = b"ACTGC";
///     let iter = NtHashIterator::new(seq, 3)?;
///     let hashes: Vec<u64> = iter.collect();
///     assert_eq!(hashes,
///                vec![0x9b1eda9a185413ce, 0x9f6acfa2235b86fc, 0xd4a29bf149877c5c]);
/// # Ok(())
/// # }
/// ```
```
(based on [this example](https://doc.rust-lang.org/stable/rustdoc/documentation-tests.html#using--in-doc-tests))

Reply:
Cool, I guess I should keep me updated with the docs :D